modules/twinkleblock.js

```diff
@@ -950,6 +950,15 @@ Twinkle.block.blockPresetsInfo = {
 		reason: '{{uw-ublock-double}} <!-- Username closely resembles another user, hard block -->',
 		summary: 'You have been indefinitely blocked from editing because your [[WP:U|username]] appears to impersonate another established Wikipedia user'
 	},
+	'uw-upeblock': {
+		autoblock: true,
+		expiry: 'infinity',
+		forRegisteredOnly: true,
+		nocreate: true,
+		pageParam: true,
+		reason: '[[WP:PAID|Undisclosed paid editing]] in violation of the WMF [[WP:TOU|Terms of Use]]',
```

Comment:
I'm mixed on the use of WP:TOU here, given that it's just a soft redirect and otherwise sparse on information.  I suppose it's what's currently being used by the template so it's fine?